### PR TITLE
Adding programming assignment for week 1 and 2

### PR DESCRIPTION
**Pull request by RajShah-1**

### CODE REVIEW

File: Programming Assignment/Week 1/RajShah_Week1/ex1data1.txt [hunk 1]
```diff
@@ -0,0 +1,97 @@
+6.1101,17.592
```
Comment (vatch123) [inline Programming Assignment/Week 1/RajShah_Week1/ex1data1.txt]: Data files are not required.

File: Programming Assignment/Week 1/RajShah_Week1/lib/jsonlab/AUTHORS.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
+The author of "jsonlab" toolbox is Qianqian Fang. Qianqian
```
Comment (vatch123) [inline Programming Assignment/Week 1/RajShah_Week1/lib/jsonlab/AUTHORS.txt]: Only source files needed.

**Review by vatch123 — CHANGES_REQUESTED** [review]

File: Programming Assignment/Week 1/RajShah_Week2/costFunction.m [hunk 3]
```diff
@@ -1,36 +0,0 @@
-function [J, grad] = costFunction(theta, X, y)
```
Comment (vatch123) [inline Programming Assignment/Week 1/RajShah_Week2/costFunction.m]: Why did you delete these files? Add all the files where you have written a code except for the warmup one

**Review by vatch123 — CHANGES_REQUESTED** [review]

File: Programming Assignment/Week 1/RajShah_Week2/costFunction.m (hunk 3 above)
Comment (RajShah-1) [reply Programming Assignment/Week 1/RajShah_Week2/costFunction.m]: By mistake I had uploaded assignment for week-2 in week-1's folder so i deleted that.
